packages/editor-ui/src/components/MultipleParameter.vue

```diff
@@ -3,10 +3,10 @@
 
 		<div class="parameter-name">
 			{{parameter.displayName}}:
-			<el-tooltip class="parameter-info" placement="top" v-if="parameter.description" effect="light">
+			<n8n-tooltip class="parameter-info" placement="top" v-if="parameter.description" >
```

Comment:
let's have the `v-if` as first attribute

Reply:
changed